hack/update-workspace-mirror.sh

```diff
@@ -29,8 +29,8 @@ BUCKET="gs://k8s-bazel-cache"
 gsutil acl get "${BUCKET}" > /dev/null
 
 tmpfile=$(mktemp bazel_workspace_mirror.XXXXXX)
-trap "rm ${tmpfile}" EXIT
-cat "$1" | while read url; do
+trap 'rm ${tmpfile}' EXIT
```

Comment:
Why did it change? What did it fix?
```
In ./hack/update-workspace-mirror.sh line 32:
trap "rm ${tmpfile}" EXIT
         ^--------^ SC2064: Use single quotes, otherwise this expands now rather than when signalled.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2064
```